### PR TITLE
Do not deduplicate for auto-complete

### LINKED ISSUE #333: Autocomplete on site field for multiple login return only one entry
For example I have many accounts on twitter, all with differents login and password on lesspass.
Is it possible to have to autocomplete box for login field so I can directly choose my different account from the same website

### PR DESCRIPTION
**Pull request by FaustXVI**
Closes #333

If this PR is ok, another one will follow in order to print the login next to the site name

### CODE REVIEW

**Comment by guillaumevincent** [comment]
Nice thank you, I will integrate this one

**Comment by FaustXVI** [comment]
Hummm... The build fails when building the container. Does it even happened? If not I'll have a look at it tomorrow.

**Comment by guillaumevincent** [comment]
@FaustXVI I will merge this pull request, failing build is not related to your patch
I will update everything to fix CI

**Comment by guillaumevincent** [comment]
If you rebase on top of master, it will work

**Comment by guillaumevincent** [comment]
I did a big refactoring on the code base.
It breaks your pull requests.

I will merge this one tomorrow
sorry

**Comment by FaustXVI** [comment]
No big deal. I'll rebase this evening.

Xavier Detant

File: package.json [hunk 1]
```diff
@@ -19,6 +19,8 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.5.4",
     "@babel/register": "^7.4.4",
+    "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-core": "7.0.0-bridge.0",
```
Comment (FaustXVI) [inline package.json]: I know you removed it but it breaks vue-jest (https://github.com/vuejs/vue-jest/issues/160).

File: package.json [hunk 2]
```diff
@@ -19,6 +19,8 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@babel/preset-env": "^7.5.4",
     "@babel/register": "^7.4.4",
+    "@vue/test-utils": "^1.0.0-beta.29",
```
Comment (FaustXVI) [inline package.json]: It seems to me that you moved all dev dependencies to the top level, so I did the same. Am I right ?

File: packages/lesspass-pure/package.json [hunk 3]
```diff
@@ -26,7 +26,15 @@
...
-    ]
+    ],
+    "moduleFileExtensions": [
+      "js",
+      "vue"
+    ],
+    "transform": {
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.js$": "../../node_modules/babel-jest"
+    }
```
Comment (FaustXVI) [inline packages/lesspass-pure/package.json]: Should I move all this to the top level ?

**Comment by guillaumevincent** [comment]
I don't know if it's a good practice or not, but some dependencies are duplicated. So yes you could move those dependencies at the root level 
`yarn add --dev -W ...`

File: package.json (hunk 1 above)
Comment (guillaumevincent) [reply package.json]: Put it back if you need it

File: package.json (hunk 2 above)
Comment (guillaumevincent) [reply package.json]: Yes you're right

File: packages/lesspass-pure/package.json (hunk 3 above)
Comment (guillaumevincent) [reply packages/lesspass-pure/package.json]: It will be ok to keep jest config in this package.json no?

**Comment by guillaumevincent** [comment]
Thank you @FaustXVI

**Pull request merged**